sus/num/__private/unsigned_integer_methods_impl.inc

```diff
@@ -748,6 +748,47 @@ _sus_pure constexpr _self _self::from_ne_bytes(
   return _self(val);
 }
 
+_sus_pure constexpr u32 _self::count_ones() const& noexcept {
+  return __private::count_ones(primitive_value);
```

Comment:
Why did these bodies need to move?

Reply:
`u32` needs to be a complete type, and `u64` is needed in `u32`, so these need to follow both types' definitions.